### PR TITLE
Tanya/add explore page

### PR DESCRIPTION
**Pull request by Tanya-atatakai**
1) add explore page, mostly copied from learn and old home page + some improvements (e.g. removing extra divs)
2) add some new chains (going to update the "add new network" doc as well)
3) update some styles on "learn" to correspond top block style with explore
4) uncomment home page button that points to explore + nav menu item

### CODE REVIEW

File: components/ExplorePage/UseCases.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+  return (
+    <div className="rounded mt-2 border p-4">
+      <div className="mb-2">
+        <h2 className="font-bold text-xl text-slate-700 align-bottom">
+          {serviceCategory.name}
+        </h2>
+      </div>
+      {filteredServices.length > 0 ? (
+        <div>
```
Comment (truemiller) [inline components/ExplorePage/UseCases.jsx:23]: could be fragment <></>

File: components/ExplorePage/UseCases.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          {filteredServices.map((service) => (
+            <div key={service.id} className="mb-2 flex items-center">
+              <a
```
Comment (truemiller) [inline components/ExplorePage/UseCases.jsx:26]: pls use next/link

File: components/ExplorePage/UseCases.jsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                  src={`/images/services/${service.iconFilename}`}
+                  width={75}
+                  height={75}
+                  alt={`${service.name} icon`}
+                  className="mr-4 hover:-translate-y-[2px] transition-transform duration-200 ease-in-out"
+                />
+              </a>
+              <div>
+                <div className="mb-1">
+                  <a
```
Comment (truemiller) [inline components/ExplorePage/UseCases.jsx:39]: next/link here too

File: components/ExplorePage/UseCases.jsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+              >
+                <Image
+                  src={`/images/services/${service.iconFilename}`}
+                  width={75}
+                  height={75}
+                  alt={`${service.name} icon`}
+                  className="mr-4 hover:-translate-y-[2px] transition-transform duration-200 ease-in-out"
+                />
+              </a>
+              <div>
```
Comment (truemiller) [inline components/ExplorePage/UseCases.jsx:37]: maybe able to remove this div

File: components/ExplorePage/UseCases.jsx [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+    buildUrl: PropTypes.string,
+    iconFilename: PropTypes.string,
+    demo: PropTypes.bool,
+    builder: PropTypes.string,
+  }).isRequired,
+};
+
+const UseCases = () => (
+  <SectionWrapper id="use-cases" customClasses="px-8 max-w-screen-xl w-full mx-auto">
+    <div className="grid grid-cols-1 gap-6 lg:grid-cols-1">
```
Comment (truemiller) [inline components/ExplorePage/UseCases.jsx:89]:
```suggestion
    <div className="grid grid-cols-1 gap-6">
```

**Review by oaksprout — COMMENTED** [review]
Reviewed, few small updates in Vercel

**Review by truemiller — DISMISSED** [review]
looks good small comments

**Review by truemiller — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]

**Pull request merged**